rpc/common/moc_common_nodeinfo.proto

```diff
@@ -61,7 +62,7 @@ message Resources {
 	reserved 4;
 	Processor processor = 1;
 	PhysicalMemory memory = 2;
-	GPU gpu = 3;	
+	GPU gpu = 3;	// delete it after gpuList is applied
 	VirtualMachineCapabilities vmCapabilities = 5;
-	repeated Gpu gpuList = 6;
+	repeated GPU gpuList = 6;
```

Comment:
I changed the GPU in node info rather than the one in compute common. Actually they have similar contents, it can be changed to  the one in compute common.